webpack.config.js

```diff
@@ -53,5 +57,8 @@ module.exports = {
             },
         ]
     },
-    plugins: [htmlPlugin, copyStatic, copyWasm]
-};
+    plugins: [htmlPlugin, copyStatic, copyWasm, new BundleAnalyzerPlugin({"openAnalyzer":false, "analyzerMode":"static"}),
+        new webpack.DefinePlugin({
+        'process.env.NODE_ENV': JSON.stringify('production')
+    }), new Visualizer({"filename":"report.html"})]
```

Comment:
what is the purpose of `report.html` ?

Reply:
To control bundle size during development, you will be able to visit it and see bundle visualization. This helped me notice that plotly.js is really big and that is why we use smaller verison which don't include some bussines charts etc

Reply:
Got it, this PR can be merged.